### ISSUE TITLE
[Feature Request]More info in application title

### ISSUE THREAD

**Issue by psc0304** [NECROBOTIO/NecroBot#2346]
Is it possible to add the info about the amount of the PokeBalls, berries and others in the application title, since runtime, lvl, XP and so are already there?
It will be better if we can customize what to display there.
And if this feature is inplemented, I don't think we need the info popping in the console anymore.

**Comment by JABirchall** [comment]
Set
      "ShowPokeballCountsBeforeRecycle": true,

**Comment by psc0304** [comment]
@JABirchall 
I mean putting the info in the title of the application, where you are see your trainer's name, runtime, XP, lvl, P/H, starbust, transferred and recycled.
All these above, and the new ones like berries, pokeballs (including totals and individual ones like GreatBall), the number of the inventory ( like 200/350), the number of the pokemon ( like 200/250) can be selected to be shown there in the title, not just keep popping in the console.

**Comment by aliceaia** [comment]
This too wordy but not necessary

**Comment by psc0304** [comment]
It can be at least optional

**Comment by nfettinger** [comment]
This can be viewed when using the visualizer. The title name is already too long to fit on the window, and the only way to view the rest is by mousing over the icon in the windows bar.
https://github.com/nicoschmitt/necrobotvisualizer/releases
